.github/actions/javascript/contributorChecklist/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Contributor Checklist Test'
+description: 'Verifies that both PR checklists are filled out'
+inputs:
+  GITHUB_TOKEN:
+    description: Auth token for New Expensify Github
+    required: true
+runs:
+  using: 'node12'
```

Comment:
Use node 16?

Reply:
Ah yes - I will update to use nvm version

Reply:
Guess we can't, I will use node16 👍